packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -590,19 +606,21 @@ export function getValidatorApi({
           blockValueEngine: `${blockValueEngine}`,
           blockValueBuilder: `${blockValueBuilder}`,
           slot,
+          shouldOverrideBuilder,
         });
       } else if (fullBlock && !blindedBlock) {
         executionPayloadSource = ProducedBlockSource.engine;
-        logger.verbose("Selected engine block: no builder block produced", {
+        logger.info("Selected engine block: no builder block produced", {
           // winston logger doesn't like bigint
           enginePayloadValue: `${enginePayloadValue}`,
           consensusBlockValueEngine: `${consensusBlockValueEngine}`,
           blockValueEngine: `${blockValueEngine}`,
           slot,
+          shouldOverrideBuilder,
```

Comment:
do we really need to include this value in the log? If it is relevant, why is it not included in the log on line 623?

Reply:
added at missing place